shacl/index.html

```diff
@@ -609,17 +609,17 @@ <h1 id="part1" style="font-size: 160%; font-weight: bold">Part 1: Core Features<
 				<ul>
 					<li><a href="#scopeNode"><b>Node scopes</b></a> define a specific RDF node as scope.</li>
 					<li><a href="#scopeClass"><b>Class-based scopes</b></a> define the scope as the set of all <span class="term"><a href="#shacl-rdfs">instances</a></span> of a class.</li><!--TODO: check instance -->
-					<li><a href="#scope"><b>General scopes</b></a> define a flexible mechanism to produce arbitrary focus nodes.</li>
+					<li><a href="#scope"><b>General scopes</b></a> are a flexible mechanism to define arbitrary focus nodes.</li>
 				</ul>
 				<p>
 					When multiple scopes are provided in a shape, the scope of a shape is the union of all focus nodes produced by these scopes.
-					Focus nodes produced by scopes may not exist in the data graph.
+					Focus nodes produced by scopes might not exist in the data graph.
 				</p>
 				
 				<section id="scopeNode">
 					<h4>Node scopes (sh:scopeNode)</h4>
 					<p>
-						Shapes define node scopes with the <code>sh:scopeNode</code> predicate.
+						Shape properties define node scopes with the <code>sh:scopeNode</code> predicate.
```

Comment:
I find the term "properties" confusing in this sentence.
it is the shape that has a scope (not the shape properties) and depending on the scope we use a different predicate (property) to define it.